src/database/models/language.py

```diff
@@ -0,0 +1,24 @@
+"""
+SQL Alchemy Table Definitions / Schema
+"""
+
+from database.manager import base_model
+from sqlalchemy import Column, Integer, String, ForeignKey, Table
+from sqlalchemy.orm import relationship
+
+
+repo_language_table = Table('RepositoryLanguages', base_model().metadata,
+    Column('repo_id', Integer, ForeignKey('repositories.id')),
+    Column('language_id', Integer, ForeignKey('languages.id'))
+)
+
+
+# ORM Class for Repositories
+class Language(base_model()):
```

Comment:
I don't disagree with your making these many-to-many relationships for language re-use .. it's a bit more expensive to do but it's right.  Can you make sure however to put a unique constraint on name here so there's only ever 1 type of a given language.